tests/test_adler32.py

```diff
@@ -0,0 +1,52 @@
+# test adler32 function
+
+import os
+import zlib
+
+import pytest
+
+import deflate
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        b"",
+        b"\0",
+        b"\0" * 23,
+        b"foobar",
+        b"foobar" * 42,
+        os.urandom(50),
+    ],
+)
+def test_adler_simple(data):
+    """test whether adler32 function computes correctly (reference: zlib.adler32)"""
+    assert deflate.adler32(data) == zlib.adler32(data)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        b"",
+        b"\0",
+        b"\0" * 23,
+        b"foobar",
+        b"foobar" * 42,
+        os.urandom(50),
+    ],
+)
+def test_adler32_with_start_value(data):
+    """test whether adler32 function computes correctly,
+    with start value (reference: zlib.adler32)"""
+    # 0 is the default start value
+    assert deflate.adler32(data, 1) == deflate.adler32(data, 1)
```

Comment:
this looks strange.

Reply:
- the ", 1" in the right call needs to be removed.
- the value in the comment / call need to be synced.

the idea of this test is to check whether the result with the start value explicitly given (but of same value as the default value) is the same as when the start value is not given.